### DISCUSSION TITLE
Problem connecting hass-music-assistant to server

### DISCUSSION THREAD

**Discussion by tenfire** [music-assistant/support#1503] [Q&A] [answered]
Hi,

I am having trouble connecting to my music assistant server from home assistant. When adding the integration I get:
```
Success!
Created configuration for Music Assistant.
```
But the integration instantly get this error:
```
Failed setup. Will retry
```

Everything else is working as it should, I can play music from music assistant web server to my local devices.

I cannot figure out what the problem is. Could it be that I'm running an unsupported version of home assistant?

```
Home Assistant version: 2023.8.2
Music Assistant server version: 2.0.0b73
hass-music-assistant version: 2023.6.0b3
```

This is my log in home assistant after adding the integration:

```
2023-08-31 17:58:20.164 INFO (MainThread) [music_assistant.client] Connected to Music Assistant Server 3af58682136a41d39240c7f1c09f5f0a using WebsocketsConnection, Version 2.0.0b73, Schema Version 23
14
2023-08-31 17:58:20.173 WARNING (MainThread) [homeassistant.config_entries] Config entry 'Music Assistant' for mass integration not ready yet: Failed to connect to music assistant server; Retrying in background
```

**Comment by Gilimi-the-Grim** [comment]
I am having maybe a similar issue.  I am running

HA 2023.8.4
HACS integration 2023.6.0b3
MA Server 2.0.0b73

I am running Home Assistant in docker attached to the host network  and I am running Music Assistant in docker on the Host network.

The web page front end seems to work.  I was able to add my Spotify Account and Local Media.

The integration gave no errors but just reads no devices or entities.  So I am wondering if this works and I have no supported player devices or is it broke?

Any thoughts?

Regards,
Joe

**Comment by tenfire** [comment, marked as answer]
I got it to work. I forgot to include the port in the base url under webservice. I had modified this field since I'm running music-assistant in kubernetes.

**Reply by Gilimi-the-Grim** [reply]
That's great news.  so did you change the port from the default of 8095?  When I installed the integration it has a default link that had the port tacked on the end.  I only changed the link to point to the IP of my MA server.  Did your players and entities show up?  My son loaned me a google mini that I will add to my HA this evening when I get home.  I am hoping my problem is that none of the current media players (roku, fully kiosk, etc) are just not supported.  Thanks for the followup.
